ecs/cluster.yaml

```diff
@@ -520,6 +520,10 @@ Resources:
           packages:
             yum:
               awslogs: []
+          # required to allow awslogs agent to pick up new threads
```

Comment:
Is this required? The jenkins template uses a similar approach where logs are inside /var/log/jenkins and it is working fine.
Why do you think this is necessary?

Reply:
Verified that it works. After +/- 2 minutes and 30 warnings the agent will pick up new threads. If you like we can get a rid of the command section.

Reply:
that would be great, thanks!